release.go

```diff
@@ -0,0 +1,104 @@
+package versionbundle
+
+import (
+	"fmt"
+
+	"github.com/coreos/go-semver/semver"
+	"github.com/giantswarm/microerror"
+)
+
+type ReleaseConfig struct {
+	Bundles []Bundle
+}
+
+func DefaultReleaseConfig() ReleaseConfig {
+	return ReleaseConfig{
+		Bundles: nil,
+	}
+}
+
+type Release struct {
+	bundles     []Bundle
+	changelogs  []Changelog
+	components  []Component
+	deprecated  bool
+	description string
+	timestamp   string
+	version     string
+}
+
+func NewRelease(config ReleaseConfig) (Release, error) {
+	if len(config.Bundles) == 0 {
+		return Release{}, microerror.Maskf(invalidConfigError, "config.Bundles must not be empty")
+	}
+
+	var changelogs []Changelog
+	var components []Component
+	var deprecated bool
+	var timestamp string
```

Comment:
These properties got added. There is more logic to be added though. I want to get the renaming and ground work done here and add more logic to aggregate the proper values in separate PRs.